etc/config/c3.amazon.properties

```diff
@@ -1,15 +1,24 @@
 compilers=&c3c
 compilerType=c3c
-defaultCompiler=c3c05
+defaultCompiler=c3c061
 supportsBinary=false
 supportsBinaryObject=false
 supportsExecute=false
-group.c3c.compilers=c3c04:c3c05
+group.c3c.compilers=c3c04:c3c050:c3c055:c3c060:c3c061
 group.c3c.isSemVer=true
 group.c3c.baseName=c3
 
-compiler.c3c04.semver=0.4
+compiler.c3c04.semver=0.4.0
 compiler.c3c04.exe=/opt/compiler-explorer/c3-0.4/c3c
 
-compiler.c3c05.semver=0.5
-compiler.c3c05.exe=/opt/compiler-explorer/c3-0.5/c3c
+compiler.c3c050.semver=0.5.0
+compiler.c3c050.exe=/opt/compiler-explorer/c3-0.5/c3c
```

Comment:
should add `compiler.c3c050.alias=c3c05`

Reply:
Updated

Reply:
Do you need anything else?

Reply:
This should be ok, thanks for the update.